components/frontend/src/header_footer/settings_menu/SettingsMenu.js

```diff
@@ -49,25 +41,19 @@ export function SettingsMenuItem({ active, children, disabled, disabledHelp, hel
             event.preventDefault()
             onClick(onClickData)
         },
+        selected: active,
         tabIndex: 0,
     }
     if (help || (disabledHelp && disabled)) {
-        props["style"] = { marginLeft: 0, marginRight: 0, marginBottom: 5 } // Compensate for the span
         return (
-            <Popup
-                content={disabledHelp || help}
-                inverted
-                position="left center"
-                // We need a span here to prevent the popup from becoming disabled when the menu item is disabled:
-                trigger={
-                    <span>
-                        <Menu.Item {...props}>{children}</Menu.Item>
-                    </span>
-                }
-            />
+            <Tooltip placement="left" title={disabledHelp || help}>
+                <span /* https://mui.com/material-ui/react-tooltip/#disabled-elements */>
```

Comment:
This comment seems unneeded here, or does it refer to the title?

Reply:
No, it explains why there is an otherwise useless span.